Service/Command.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Zumba\User\Service;
+
+use \Zumba\User\Contract\Command as CommandInterface,
+	\Zumba\User\Contract\Transactional;
+
+abstract class Command implements CommandInterface, Transactional {
```

Comment:
Where is the definition of CommandInterface? I thought we were just going to call it Command instead of CommandInterface.

Reply:
@bgarb002 

```use \Zumba\User\Contract\Command as CommandInterface,```